src/engine/sys/sdl_input.cpp

```diff
@@ -545,9 +545,9 @@ static void IN_InitJoystick()
 	{
 		Log::Debug( "Calling SDL_Init(SDL_INIT_JOYSTICK)..." );
 
-		if ( SDL_Init( SDL_INIT_JOYSTICK ) == -1 )
+		if ( SDL_Init( SDL_INIT_JOYSTICK ) < 0 )
 		{
-			Log::Debug( "SDL_Init(SDL_INIT_JOYSTICK) failed: %s", SDL_GetError() );
+			Log::Notice( "SDL_Init(SDL_INIT_JOYSTICK) failed: %s", SDL_GetError() );
```

Comment:
Would it error out if there is no joystick, hence the non-usage of `Warn` to avoid an useless warning if something not required is missing ?